client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -169,6 +169,7 @@ class StatsPostDetail extends Component {
 
 		// Prepare post details to PostStatsCard from post or postFallback.
 		const passedPost = this.getPost();
+		passedPost.url = previewUrl;
```

Comment:
I'm reusing the URL that's already available in the post detail page and forwarding it.